Pizzeria/Pizzeria/Employee/Administrator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Pizzeria
+{
+    public class Administrator : Employee
+    {
+        private List<Employee> _employees = new List<Employee>();
```

Comment:
Some strange design. With this, I can add Administrator as an employee of the administrator